RELEASES.md

```diff
@@ -0,0 +1,36 @@
+# CITGM Release Process
+
+This document describes the technical aspects of the citgm release process. The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.
+
+## Who can make a release?
+
+Any member of the @nodejs/citgm team can make a release.
```

Comment:
I was imagining that with this language we would begin giving the publish bit to anyone who is part of @nodejs/citgm